beetsplug/lastgenre/__init__.py

```diff
@@ -179,12 +201,18 @@ def _resolve_genres(self, tags):
                     parents = [find_parents(tag, self.c14n_branches)[-1]]
 
                 tags_all += parents
-                if len(tags_all) >= count:
+                # Don't break here if we need to sort by specificity, which happens with
+                # the full tag list below
```

Comment:
It can be helpful to phrase comments in a positive way. For example, consider "Stop if we have enough tags already, unless we need to find the most specific tag (instead of the most popular)."